### PR TITLE
translate several articles into Chinese

### PR DESCRIPTION
**Pull request by randomwalk266**

### CODE REVIEW

**Comment by randomwalk266** [comment]
@bobdos

**Comment by bobdos** [comment]
will check asap.

File: i18n/zh-cn.toml [hunk 1]
```diff
@@ -108,181 +108,181 @@ other = "y"
...
 
 ["wallets[multidoge][title1]"]
 other = "MultiDoge"
 
 ["wallets[multidoge][text1]"]
-other = "MultiDoge is a 'light' wallet. It syncs with the blockchain by 'skimming' through the blockchain, providing fast sync times."
+other = "MultiDoge是一个轻量级钱包。它通过“跳读”浏览区块链网络来进行数据同步，提高了同步的效率。"
 
 ["wallets[core][title1]"]
 other = "Dogecoin Core"
```
Comment (bobdos) [inline i18n/zh-cn.toml:187]: Dogecoin Core -> 狗狗币核心

File: i18n/zh-cn.toml [hunk 2]
```diff
@@ -108,181 +108,181 @@ other = "y"
...
 
 ["wallets[core][text1]"]
-other = "Dogecoin Core, on the other hand, is a 'full' wallet. It syncs by downloading the blockchain, providing a solid-working Dogecoin wallet."
+other = "Dogecoin Core是一个完整的钱包，它通过下载完整的区块链数据来进行同步，提供了一个独立运行的狗狗币钱包。"
```
Comment (bobdos) [inline i18n/zh-cn.toml:190]: Dogecoin Core -> 狗狗币核心

File: i18n/zh-cn.toml [hunk 3]
```diff
@@ -108,181 +108,181 @@ other = "y"
...
-other = "Foundation"
+other = "基金会"
 
 ["resources[text1]"]
-other = "Learn more about the Dogecoin Foundation and its Trailmap for Dogecoin!"
+other = "了解更多关于狗狗币基金会和狗狗币路线图！"
 
 ["resources[title2]"]
-other = "Blogs"
+other = "博文"
```
Comment (bobdos) [inline i18n/zh-cn.toml:230]: 博文 -> 博客

File: i18n/zh-cn.toml [hunk 4]
```diff
@@ -108,181 +108,181 @@ other = "y"
...
 
 ["resources[text2]"]
-other = "Stay up to date with blog posts from Foundation Developers!"
+other = "随时了解基金会开发者们的博文！"
```
Comment (bobdos) [inline i18n/zh-cn.toml:233]: 博文 -> 博客

**Review by bobdos — CHANGES_REQUESTED** [review]

**Comment by randomwalk266** [comment]
@bobdos

**Review by bobdos — APPROVED** [review]
all good and it's ready to go!

many thanks.

**Pull request merged**